app/models/coronavirus/sub_section.rb

```diff
@@ -4,11 +4,17 @@ class Coronavirus::SubSection < ApplicationRecord
   belongs_to :page, foreign_key: "coronavirus_page_id"
   validates :title, :content, presence: true
   validates :page, presence: true
-  validate :action_link_url_must_be_in_content
+  validate :all_action_link_fields_must_exist_or_not
 
-  def action_link_url_must_be_in_content
-    if action_link_url.present? && !content.include?(action_link_url)
-      errors.add(:action_link_url, "does not exist in accordion content")
+  def all_action_link_fields_must_exist_or_not
```

Comment:
I wonder if this can be simplified with:

```
return if action_link_url.present? && action_link_content.present? && action_link_summary.present?
return if action_link_url.blank? && action_link_content.blank? && action_link_summary.blank?

errors.add(:action_link_url, "is blank") if action_link_url.blank?
errors.add(:action_link_content, "is blank") if action_link_content.blank?
errors.add(:action_link_summary, "is blank") if action_link_summary.blank?
```